src/main/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCM.java

```diff
@@ -55,17 +53,14 @@
 import java.util.logging.Logger;
 import java.util.stream.Collectors;
 
-import static com.atlassian.bitbucket.jenkins.internal.credentials.BitbucketCredentialsAdaptor.createWithFallback;
+import static com.atlassian.bitbucket.jenkins.internal.credentials.CredentialUtils.getCredentials;
```

Comment:
Now that this is statically imported, there are a bunch of `CredentialUtils.getCredentials` calls that can be unqualified

Reply:
This is now changed to the version which is based on your branch. this is not there anymore